### DISCUSSION TITLE
prescription drug file plan definition

### DISCUSSION THREAD

**Discussion by atw1223a** [CMSgov/price-transparency-guide#107] [Q&A] [answered]
what criteria exactly defines 'a plan'? is it an offering to a group? or a plan specification? or is it a formulary? or a copay structure?

for example, if a client offers 5 copay structures with 1 formulary, is it one file per plan? or 1 file because all 5 copay structures utilize the same formulary and the per unit pricing would be the same?

**Comment by shaselton-usds** [comment, marked as answer]
@atw1223a wrapping up loose ends - as you probably are aware, enforcement for the Rx file has been deferred. No need to worry about producing that file at this moment.
